examples/secp256k1/README.md

```diff
@@ -2,22 +2,26 @@
 
 The spore secp256k1 examples is a collection of code examples written in TypeScript, intended to help developers understand how to use `@spore-sdk/core` in Node environment to:
 
-- Construct and send transactions of spores and clusters on Testnet (Aggron)
-- Use Secp256k1Blake160 lock to sign transactions
+- Construct and send transactions of spores/clusters on Testnet (Aggron)
+- Use Secp256k1Blake160 lock for transactions
 
 ## Run examples
 
+> Please make sure you've met the pre-requirements:  
+> `Node.js` >= 18.0.0  
+> `PNPM` >= 8.0.0  
+
 ### Setup
 
-Let's run the spore secp256k1 examples, but first we need to set up the local environment. 
+Ensure to configure the local environment before running the examples.
+
 Assuming you've cloned the repo, and your local `spore-sdk` is located at:
```

Comment:
Once you have cloned the repository, ensure that your local `spore-sdk` is located at: